### PR TITLE
The Ember Times - Issue No. 77

### PR DESCRIPTION
**Pull request by amyrlam**
Ideas, feel free to add to list or claim!

- [ ] EmberCamp talks, deep dive? http://embercamp.com/ and https://github.com/ember-chicago/ember-camp-2018-notes Maybe we wait for all talk videos to be published? Keep an eye on #ember-camp in Discord.
- [x] New RFC: emberjs/rfcs#407 (🔏 @jessica-jordan)
- [ ] New RFC: emberjs/rfcs#403 (still WIP, might be good to check with @runspired if it's ready for Ember Times)
- [ ] Check https://github.com/jessica-jordan/whats-new-in-emberland (roundup of all the PRs in emberjs and ember-learn repos)
- [ ] https://github.com/microstates/ember, demos https://ember-microstates.netlify.com/
- [x] Components patterns in Ember.js - https://twitter.com/TheMacsour/status/1072088269742112768 (🔒 @chrisrng )
- [ ] mention npm survey again, with last year's results showing ember rising (and allude to state of js lack of representation?) https://twitter.com/seldo/status/1072619425537368064
- [x] Emberconf
- [x] Ember 3.6 released (🔏@kennethlarsen)

Todos:

- [ ] Goodbits: test in Mac mail, weird formatting issue with intro last time
- [ ] Ensure display on emberjs.com/blog is good e.g. buttons
- [ ] Document https://github.com/jessica-jordan/emberjs-times-tools?
* Have to move social links to the bottom (above footer)
* Delete extra blank line at the end of each section
* Replace broken image in Readers' Questions with https://emberjs.com/images/tomsters/officehours-42680347.png
- [ ] Set up tweet for Monday morning
- [ ] Set $natural reminder in channel for Alon to post Monday morning if avail?

### CODE REVIEW

**Comment by tylerturdenpants** [comment]
Issues that arise via goodbits, can we please open an issue on the import script?

**Comment by amyrlam** [comment]
@tylerturdenpants sure thing, what I listed above is more existing things to keep an eye on. Excited to try out your new script!

File: source/blog/2018-12-14-the-ember-times-issue-77.md [hunk 1]
```diff
@@ -25,8 +25,10 @@ Leave your thoughts on how boolean arguments could be implemented for Ember comp
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Ember 3.6 released 🚀](https://emberjs.com/blog/2018/12/13/ember-3-6-released.html)
+Ember 3.6 is out and ready to use. This release contains the last step of the **Native Class Constructor Update RFC**.  This introduces native classes in Ember but with some caveats so remember to read the [release post]([Ember.js - Ember 3.6 Released](https://emberjs.com/blog/2018/12/13/ember-3-6-released.html#toc_new-features-2)) carefully before refactoring your code base.
```
Comment (jessica-jordan) [inline source/blog/2018-12-14-the-ember-times-issue-77.md]: I think the link for the release post is somewhat doubled regarding formatting. Would it be

```
[release post](https://emberjs.com/blog/2018/12/13/ember-3-6-released.html#toc_new-features-2)
```

File: source/blog/2018-12-14-the-ember-times-issue-77.md [hunk 2]
```diff
@@ -25,8 +25,10 @@ Leave your thoughts on how boolean arguments could be implemented for Ember comp
...
 
+Ember 3.6 also contains the final stage of the **router service RFC** which includes some new methods that can improve the way you deal with routing. There’s also new deprecations and some small fixes in Ember CLI so be sure to check out the complete [release post]([Ember.js - Ember 3.6 Released](https://emberjs.com/blog/2018/12/13/ember-3-6-released.html))
```
Comment (jessica-jordan) [inline source/blog/2018-12-14-the-ember-times-issue-77.md]: I think the same would apply here:

```
[release post](https://emberjs.com/blog/2018/12/13/ember-3-6-released.html#toc_new-features-2)
```

**Pull request merged**